dataland-community-manager/src/main/kotlin/org/dataland/datalandcommunitymanager/model/dataRequest/StoredDataRequestMessageObject.kt

```diff
@@ -0,0 +1,9 @@
+package org.dataland.datalandcommunitymanager.model.dataRequest
+
+data class StoredDataRequestMessageObject(
+    var contactList: List<String>? = null,
+
+    val message: String? = null,
+
+    val updateTimestamp: Long? = null,
```

Comment:
Is it update or something like lastModifiedTimestamp?

Reply:
Is an update not a modification? :) It should indicate the last change of the message object attributes.